core/include/misc/status.h

```diff
@@ -3,9 +3,10 @@
  *
  * @section LICENSE
  *
- * The MIT License
+ * The BSD License
```

Comment:
The section below `@copyright` must change, as this text is specific to the MIT license. We need to find the corresponding one for the BSD license.